mmv1/templates/tgc_next/test/test_file.go.tmpl

```diff
@@ -15,12 +15,16 @@
 package {{$.PackageName}}_test
 
 import (
+	"os"
 	"testing"
 
 	"github.com/GoogleCloudPlatform/terraform-google-conversion/v7/test"
 )
 
 func TestAcc{{$.ResourceName}}(t *testing.T) {
+	if os.Getenv("WRITE_FILES") != "" {
+		t.Parallel()
```

Comment:
I recall we had a RAM problem recently, do we have sufficient RAM to run the tests in parallel?

Reply:
With `WRITE_FILES` set, the tests are only run in parallel locally. It will not affect the tests running in CI.